citus_dev/citus_dev

```diff
@@ -160,17 +154,12 @@ def main(arguments):
                 nodeport = port + i
                 run(f'psql -p {nodeport} -c "CREATE EXTENSION citus;"')
 
-            # If the cluster size is 0 we add the coordinator as the only node, otherwise we will add all other nodes
-            if size == 0:
-                run(f"psql -p {port} -c \"SELECT * from master_add_node('localhost', {port}, groupId => 0);\"")
-            else:
-                for i in range(size):
-                    workerport = port + 1 + i
-                    run(f"psql -p {port} -c \"SELECT * from master_add_node('localhost', {workerport});\"")
-
-                    if arguments["--mx"]:
-                        run(f"psql -p {port} -c \"SELECT start_metadata_sync_to_node('localhost', {workerport});\"")
+            # we always add the coodinator
```

Comment:
I think this comment only makes sense in the context of mx. To reduce confusion I would _not_ have any comment here.

Reply:
makes sense